e2e/test/provisioning/ProvisioningE2ETests.cs

```diff
@@ -45,6 +45,7 @@ public class ProvisioningE2ETests : E2EMsTestBase
 
         private static DirectoryInfo s_x509CertificatesFolder;
         private static string s_intermediateCertificateSubject;
+        private static CancellationTokenSource s_cancellationTokenSource = new CancellationTokenSource();
```

Comment:
Why do this instead of just passing `CancellationToken.None` in the tests?

This type is IDisposable, too.